portfolio/src/main/webapp/script.js

```diff
@@ -32,3 +32,14 @@ function addFunFacts() {
     const factsContainer = document.getElementById('facts-container');
     factsContainer.innerText = fact;
 }
+
+
+/** Fetches the current date from the server and adds it to the page. */
+async function showServerTime() {
+  const responseFromServer = await fetch('/date');
+//   const responseFromServer = await fetch('Saturday 6th March, 2021');
```

Comment:
Commented out code should generally be removed and not submitted as part of your PRs.

Reply:
Okay, I will work on that now.